### DISCUSSION TITLE
MLFlow logger

### DISCUSSION THREAD

**Discussion by Jeronymous** [speechbrain/speechbrain#2363] [Feature Request]
### 🚀The feature

Several loggers are implemented in Speechbrain, including Tensorboard.
It would be good to have MLFlow as well.
Has it ever been considered?

### Solution outline

It could be a class `speechbrain.utils.train_logger.MLFlowLogger` that would be initialized with an experiment name.

Similarly to what is done in `transformers` : https://julsimon.medium.com/using-mlflow-with-hugging-face-transformers-4f69093a6c04

**Comment by Adel-Moumen** [comment]
Hi @Jeronymous, thanks again for improving SpeechBrain!

Hi @pplantinga @TParcollet @mravanelli, what do you think about adding MLFlow in SB?
